app/uk/gov/hmrc/plasticpackagingtax/registration/forms/CommonFormValidators.scala

```diff
@@ -51,7 +51,7 @@ trait CommonFormValidators {
   val isValidEmail: String => Boolean = (email: String) =>
     email.isEmpty || isMatchingPattern(email, emailPattern)
 
-  val phoneNumberRegexPattern: Pattern = Pattern.compile("^[\\d ]*$")
+  val phoneNumberRegexPattern: Pattern = Pattern.compile("^[+]?[0-9 ]*$")
```

Comment:
We did discuss whether hyphen and brackets might be permitted?

Reply:
The discussion of whether - and brackets should be permitted is yet to be discussed with content guys.

Reply:
Do we know whether the + character will be permitted by ETMP?

Reply:
should be - the regex in sub create call allows it

Reply:
ETMP defines the field as `CHAR 24` but might be worth checking with Andrew Blackman?

Reply:
The phone number pattern in API spec is '^[0-9 )/(*#+-]+$' so it allows + - brackets

Reply:
I know that this will be permitted by EIS, I just thought it worth checking that the real backend (ETMP) is cool with it. I'd be surprised if it's not.